Fixes #9550 - Refresh samples and note supported platforms (#9561)
MicrosoftDocs/PowerShell-Docs: reference/docs-conceptual/samples/Manipulating-Items-Directly.md

You can use the same technique to create a new registry key. In fact, a registry key is easier to
create because the only item type in the Windows registry is a key. (Registry entries are item

> **properties**.) For example, to create a key named `_Test` in the `CurrentVersion` subkey, type:

Reviewer: ```suggestion
_properties_.) For example, to create a key named `_Test` in the `CurrentVersion` subkey, type:
```

To move a file or folder, use the `Move-Item` cmdlet.

For example, the following command moves the New.Directory directory from the `C:\temp` directory to
the root of the `C:` drive. To verify that the item was moved, include the **PassThru** parameter of

> the `Move-Item` cmdlet. Without **Passthru**, the `Move-Item` cmdlet doesn't display any results.

Reviewer: ```suggestion
the `Move-Item` cmdlet. Without **PassThru**, the `Move-Item` cmdlet doesn't display any results.
```

## Copying Items (Copy-Item)

If you are familiar with the copy operations in other shells, you might find the behavior of the
`Copy-Item` cmdlet in PowerShell to be unusual. When you copy an item from one location to another,

> Copy-Item doesn't copy its contents by default.

Reviewer: ```suggestion
`Copy-Item` doesn't copy its contents by default.
```

## Deleting Items (Remove-Item)

To delete files and folders, use the `Remove-Item` cmdlet. PowerShell cmdlets, such as
`Remove-Item`, that can make significant, irreversible changes will often prompt for confirmation
when you enter its commands. For example, if you try to remove the `New.Directory` folder, you will
be prompted to confirm the command, because the folder contains files:

```powershell
Remove-Item C:\temp\New.Directory
```

```Output
Confirm

> The item at C:\temp\New.Directory has children and the -recurse parameter wasn't

Reviewer: ```suggestion
The item at C:\temp\New.Directory has children and the -recurse parameter was not
```
